.github/workflows/ci_tests.yml

```diff
@@ -131,12 +131,19 @@ jobs:
   pytest-real-backend:
     runs-on: ubuntu-latest
     name: pytest-real backend
-    # timeout-minutes: 15
 
     env:
       PORT: 8080
       OTEAPI_prefix: "/api/v1"
 
+    services:
```

Comment:
Or I guess maybe the error was this service wasn't there?

Reply:
The error was the use of `docker-compose` in this job, which the GitHub Actions CI framework has some issues with, apparently. Instead I now use `docker`.